### DISCUSSION TITLE
the cmd to install all extension

### DISCUSSION THREAD

**Discussion by sky5454** [coder/code-server#5912] [Ideas]
I use those cmd to install all extensions but only installed one 
```sh
code-server --install-extension *.vsix
```

so I think if it could be improved so that use those cmd to install all extensions.

**Comment by jsjoeio** [comment]
Feel free to search and see if there's an issue open, otherwise you can open a new one!

**Comment by Jayball2052** [comment]
I want to unsubscribe

**Reply by lg-nineoneninethreenine** [reply]
Hit the button in the desktop website, you can unsubscribe.

**Reply by lg-nineoneninethreenine** [reply]
Or just edit your watching settings.
